x-pack/test/api_integration/apis/ml/annotations/update_annotations.ts

```diff
@@ -134,17 +105,23 @@ export default ({ getService }: FtrProviderContext) => {
       expect(body.result).to.eql('updated');
 
       const updatedAnnotation = await ml.api.getAnnotationById(originalAnnotation._id);
-      expect(updatedAnnotation?.annotation).to.eql(annotationUpdateRequestBody.annotation);
-      expect(updatedAnnotation?.detector_index).to.eql(annotationUpdateRequestBody.detector_index);
-      expect(updatedAnnotation?.event).to.eql(annotationUpdateRequestBody.event);
+      if (updatedAnnotation) {
+        Object.keys(commonAnnotationUpdateRequestBody).forEach((key) => {
+          const field = key as keyof Annotation;
+          expect(updatedAnnotation[field]).to.eql(annotationUpdateRequestBody[field]);
+        });
+      }
     });
 
     it('should not update annotation for unauthorized user', async () => {
       const annotationsForJob = await ml.api.getAnnotations(jobIds[2]);
       expect(annotationsForJob).to.have.length(1);
 
       const originalAnnotation = annotationsForJob[0];
+
       const annotationUpdateRequestBody = {
+        timestamp: Date.now(),
+        end_timestamp: Date.now(),
```

Comment:
Is there a reason why you did not use `...commonAnnotationUpdateRequestBody,` in this case?